manifests/post-installation/hbn-template.yaml

```
@@ -0,0 +1,19 @@
+  helmChart:
+    source:
+      repoURL: https://helm.ngc.nvidia.com/nvidia/doca
+      version: "1.0.2"
+      chart: doca-hbn
+    values:
+      image:
+        repository: quay.io/itsoiref/hbn
+        tag: 25.4-doca-3.0-1
+      resources:
+        memory: 6Gi
+        nvidia.com/bf_sf: 3
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)